watcher/src/watcher_db.rs

```diff
@@ -456,7 +456,11 @@ impl WatcherDB {
                 Ok((ts, res)) => match res {
                     TimestampResultCode::WatcherBehind => {
                         if watcher_behind_timer.elapsed() > watcher_timeout {
-                            log::warn!(self.logger, "watcher is still behind on block index = {} after waiting {} seconds, caller will be blocked", block_index, watcher_timeout.as_secs());
+                            log::warn!(
+                                self.logger,
+                                "Caller will be blocked in {} seconds",
```

Comment:
Do we lose some context there about why "Caller will be blocked?"  I think the original message seems to have more human context.

Reply:
I'll change it back